src/test/java/com/alkemy/ong/service/impl/ActivityServiceImplTest.java

```diff
@@ -0,0 +1,168 @@
+import com.alkemy.ong.model.Activity;
+import com.alkemy.ong.model.Testimonial;
+import com.alkemy.ong.repository.ActivityRepository;
+import com.alkemy.ong.service.ActivityService;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.modelmapper.ModelMapper;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.boot.test.mock.mockito.SpyBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.validation.BindingResult;
+
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.*;
+
+@ContextConfiguration(classes = {ActivityServiceImpl.class})
+@ExtendWith(SpringExtension.class)
+@DisplayNameGeneration(value = DisplayNameGenerator.ReplaceUnderscores.class)
+class ActivityServiceImplTest {
+
+    @SpyBean
+    private ModelMapper modelMapper;
+
+    @MockBean
+    private ActivityRepository activityRepository;
+
+    @MockBean
+    private ActivityService activityService;
+    private ActivityServiceImpl activityServiceImpl;
+
+    @BeforeEach
+    void setUp() {
+        activityServiceImpl = new ActivityServiceImpl( activityRepository, modelMapper );
+    }
+
+    private static ActivityDto getActivityDto() {
+
+        ActivityDto activityDto = new ActivityDto();
+        activityDto.setContent( "Not all who wander are lost" );
+        activityDto.setImage( "Image" );
+        activityDto.setName( "Name" );
+        return activityDto;
+    }
+
+    private Activity getActivityWithId() {
+
+        Activity activity = getActivity( false );
+        activity.setId( "algo" );
+        return activity;
+    }
+
+    private static Activity getActivity(boolean softDelete) {
+
+        Activity activity = new Activity();
+        activity.setContent( "Not all who wander are lost" );
+        activity.setId( "42" );
+        activity.setImage( "Image" );
+        activity.setName( "Name" );
+        activity.setDeleted( softDelete );
+        activity.setTimestamp( Timestamp.from( LocalDateTime.of( 1, 1, 1, 1, 1, 1 ).toInstant( ZoneOffset.UTC ) ) );
+        return activity;
+    }
+
+    private Activity getUpdatedTestimonial() {
+
+        Activity activityWithId = getActivityWithId();
+        activityWithId.setId( "42" );
+        activityWithId.setName( "updated" );
+        activityWithId.setImage( "updated" );
+        activityWithId.setContent( "updated" );
+        activityWithId.setDeleted( false );
+        return activityWithId;
+    }
+
+    @Test
+    void UpdateActivityAndReturnEntityUpdated() {
```

Comment:
Como aca no usas `@DisplayName`, y ya tenes la anotacion para reemplazar los guion bajo, te conviene cambiar como ejemplo este nombre a `Update_activity_and_return_entity_updated` y quedaria `void Update_activity_and_return_entity_updated() `
```suggestion
    void Update_activity_and_return_entity_updated() {
```
Los mismo con los demas nombres.

Reply:
También cambio todo acá al formato correcto, gracias!